src/Management/test/Endpoint.Test/Refresh/EndpointMiddlewareTest.cs

```diff
@@ -60,6 +60,8 @@ public async Task RefreshActuator_ReturnsExpectedData()
         string ancEnv = Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT");
         Environment.SetEnvironmentVariable("ASPNETCORE_ENVIRONMENT", null);
 
+        Environment.SetEnvironmentVariable("ASPNETCORE_URLS", null);
```

Comment:
Is this needed due to other tests leaking?

Reply:
Yes.